examples/k8s/manifest.yaml

```diff
@@ -63,7 +63,7 @@ spec:
     spec:
       containers:
         - name: everything
-          image: ghcr.io/agentgateway/agentgateway:latest-ext
+          image: ghcr.io/agentgateway/agentgateway:0.4.21-ext
```

Comment:
I added this tag to the latest release, so this is no longer necessary: https://github.com/agentgateway/agentgateway/pkgs/container/agentgateway/402138852?tag=latest-ext

Reply:
oh GREAT! updated PTAL @EItanya